client/app/hearings/components/AddHearingDay.jsx

```diff
@@ -213,86 +233,135 @@ export const HearingDayAddModal = ({
-              onChange={(value, label) => props.selectHearingCoordinator({ value, label })} />
-          </React.Fragment>
-          }
-          <TextareaField
-            name="Notes (Optional)"
-            strongLabel
-            onChange={(value) => props.setNotes(value)}
-            textAreaStyling={notesFieldStyling}
-            value={notes} />
+            options={filteredRequestTypeOptions(REQUEST_TYPE_OPTIONS)}
+          />
           <Checkbox
             name="roomRequired"
-            label="Assign Board Hearing Room"
+            label="Assign a Board hearing room"
             disabled={selectedVirtual}
             strongLabel
             value={selectedVirtual ? false : roomRequired}
             onChange={(value) => props.onAssignHearingRoom(value)}
-            {...roomRequiredStyling} />
+            {...roomRequiredStyling}
+          />
+          {(selectedVideo || selectedVirtual) && (
+            <RegionalOfficeDropdown
+              label="Regional Office (RO)"
+              excludeVirtualHearingsOption={!selectedVirtual}
+              errorMessage={errorMessages?.ro ? getErrorMessage(true) : null}
+              onChange={onRoChange}
+              value={selectedRegionalOffice?.key}
+            />
+          )}
+          {selectedRequestType !== null && (
+            <React.Fragment>
+              <JudgeDropdown
+                optional
+                name="vlj"
+                label="VLJ"
+                value={vlj?.value}
+                onChange={(value, label) => props.selectVlj({ value, label })}
+              />
+              <HearingCoordinatorDropdown
+                optional
+                name="coordinator"
+                label="Hearing Coordinator"
+                value={coordinator?.value}
+                onChange={(value, label) => props.selectHearingCoordinator({ value, label })}
+              />
+            </React.Fragment>
+          )}
+          <TextareaField
+            optional
+            name="Notes"
+            strongLabel
+            onChange={(value) => props.setNotes(value)}
+            textAreaStyling={notesFieldStyling}
+            value={notes}
+          />
+          {showTimeSlots && (
+            <React.Fragment>
+              <div className="cf-help-divider usa-width-one-whole" />
+              <TimeSlotCount onChange={(value) => handleSlotCountChange(value)} value={slotCount} />
+              <TimeSlotLength onChange={(value) => handleSlotLengthChange(value)} value={slotLength} />
+              <HearingTime
+                disableRadioOptions
+                regionalOffice={selectedRegionalOffice?.key}
+                vertical
+                label="Start Time of Slots"
+                enableZone
+                localZone="America/New_York"
+                onChange={handleStartTimeChange}
+                value={hearingStartTime}
+              />
+              {hearingStartTime && <div className="time-slot-preview-container">
```

Comment:
```suggestion
              <div className="time-slot-preview-container">
```
This is no longer neeeded per this [commit](https://github.com/department-of-veterans-affairs/caseflow/pull/16288/commits/062b016ecfa895a7d7b31b0a4a33491e0948a9e8).